lib/scrub/scrub_runner.rb

```diff
@@ -174,6 +178,25 @@ def download_to_work_dir(file)
 
     private
 
+    def check_new_types(files)
+      # Get the types of the files we are loading.
+      types = []
```

Comment:
Any reason not to use a `Set` for this, and either `to_a` it before passing it to the TypeChecker, or even have TypeChecker take sets in its initializer? You want to sort them only for display, and that's the edge case.

Reply:
Actually swapping in Set works just fine, so I did.